deploy/kind/dbsp-deploy.yml

```diff
@@ -57,7 +57,7 @@ spec:
         app: dbsp
     spec:
       containers:
-      - image: ghcr.io/feldera/dbsp-manager
+      - image: ghcr.io/feldera/feldera-manager
```

Comment:
not sure if that one is right?

Reply:
Good catch. Thanks. Fixed.

`git grep 'feldera/feldera[^./">,)]'` shows that this was probably the only one.